tracee-ebpf/tracee/tracee.go

```diff
@@ -905,7 +905,7 @@ func (t *Tracee) Close() {
 	if t.bpfModule != nil {
 		t.bpfModule.Close()
 	}
-	t.printer.Close()
+	//t.printer.Close()
```

Comment:
Any reason for that?

Reply:
accidentally added this here. it's not related to this PR